fixtures/eurolite/edx-4.json

```
@@ -21,32 +22,20 @@
     "power": 3680,
     "DMXconnector": "3-pin"
   },
-  "availableChannels": {
-    "Dimmer (All Channels)": {
-      "defaultValue": 0,
-      "capability": {
-        "type": "Intensity"
-      }
-    },
-    "Dimmer (Channel 1)": {
-      "defaultValue": 0,
-      "capability": {
-        "type": "Intensity"
-      }
-    },
-    "Dimmer (Channel 2)": {
-      "capability": {
-        "type": "Intensity"
-      }
-    },
-    "Dimmer (Channel 3)": {
-      "capability": {
-        "type": "Intensity"
-      }
-    },
-    "Dimmer (Channel 4)": {
+  "matrix": {
+    "pixelCount": [4, 1, 1],
+    "pixelGroups": {
+      "Master": "all",
+      "1/2": ["1", "2"],
+      "3/4": ["3", "4"]
```

Those pixel groups are not needed here

```suggestion
      "Master": "all"
```